src/commands/index.ts

```diff
@@ -3,8 +3,10 @@ import { IndexAdapter } from "../index/index-adapter";
 import { Command, CommandType } from "./command";
 
 export class IndexCommand extends Command {
+  public static readonly CommandName = "index-document";
```

Comment:
I went with this approach as I needed a way to get the command name without registering the command with vscode.

If I did `new IndexCommand().command()` this would register the command into `vscode` as the constructor does this work. When wiring up the placeholder commands I need a way to get the command name without wiring up the command. 

For example:

`Command.dynamicRegister(IndexCommand.CommandName, IndexerNotEnabledCommandHandler);`